autograder/cli/users/auth.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+import autograder.api.users.auth
+
+def run(arguments):
+    try:
+        result = autograder.api.users.auth.send(arguments, exit_on_error = False)
+    except Exception as ex:
+        if ex.code != 401:
```

Comment:
Use the constant from `http.HTTPStatus` instead of the raw number.

Applies to multiple places in this PR.